modules/mobile/app/controllers/mobile/v0/facility_eligibility_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class FacilityEligibilityController < ApplicationController
+      DEFAULT_PAGE_NUMBER = 1
+      DEFAULT_PAGE_SIZE = 3
+      SERVICE_TYPES = %w[primaryCare nutrition podiatry optometry audiology].freeze
+
+      def index
+        require_params
```

Comment:
Is there a better way to do this?

Reply:
One option I've seen is to create individual methods for the variables like:
```
def service_type
  params.require(:serviceType)
end
```
Then use those variables instead of directly calling the params. Alternatively, you could use a before_action, which is pretty much equivalent to what you're doing. I would probably lean more toward creating methods, unless you're doing expensive things before the methods would be called.

Reply:
Yes, I like that better.